src/vt100_imp.rs

```diff
@@ -37,8 +37,11 @@ impl Cell for vt100::Cell {
 fn fill_buf_cell(screen_cell: &vt100::Cell, buf_cell: &mut ratatui::buffer::Cell) {
     let fg = screen_cell.fgcolor();
     let bg = screen_cell.bgcolor();
-
-    buf_cell.set_symbol(&screen_cell.contents());
+    if screen_cell.has_contents() {
+        buf_cell.set_symbol(&screen_cell.contents());
+    } else {
+        buf_cell.set_symbol(" ");
```

Comment:
Did you find this is needed? 

I removed  `buf_cell.set_symbol(" ");` and saw no noticeable difference.

Reply:
> Did you find this is needed?
> 
> I removed `buf_cell.set_symbol(" ");` and saw no noticeable difference.

Fixed. On a second thought I think it's not needed.